osin/install.go

```diff
@@ -0,0 +1,26 @@
+package osin
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	osinv1 "github.com/openshift/api/osin/v1"
+)
+
+const (
+	GroupName = "osin.config.openshift.io"
```

Comment:
@deads2k i asked @sttts what `osin` means and he replied with "I asked the same question"... two people asking what this means smells like "everybody will ask what this is"... maybe we should spend some time bike-shedding the name here?

Reply:
oauth-server or something like that would be better.

Reply:
yeah, I like oauth-server (at least I know what this is ;-)

Reply:
or osin-oauth-server if we want to have osin in the name because it is too specific.

Reply:
@sttts @mfojtik this config belongs to sig-security.  This is the name they chose.

Reply:
/cc @openshift/sig-security